src/sim/sim_driver.cc

```diff
@@ -181,6 +181,51 @@ class SRAM {
     }
     memset(sram_ptr, 0, kElemBytes * xtotal * op->y_pad_1);
   }
+
+  // This is for load 8bits to ACC only
+  void Load_int8(const VTAMemInsn* op,
+            DRAM* dram,
+            uint64_t* load_counter,
+            bool skip_exec) {
+    CHECK_EQ(kBits, VTA_ACC_WIDTH);
+
+    // TODO(zhanghao): extend to other width
+    CHECK_EQ(VTA_ACC_WIDTH, 32);
+    CHECK_EQ(VTA_INP_WIDTH, 8);
```

Comment:
Should we include an assertion for the WGT width as well?

Reply:
This code section is for the newly added load_int8 instructions in fsim. The instruction will load and cast 8-bit inputs from input memory and copy onto the 32-bit acc memory. As of now, this instruction only support 8-bit source and 32-bit acc mem as destination, although we may expend the instruction further.
As this instruction does not operate on wgt_mem, no requirement on wgt width is specified here.

On the other hand, for our current intelfocl/vitis implementation, we do have a restriction on the width of wgt mem, and we have already added an assertion there.

Reply:
Makes total sense thank you